### ISSUE TITLE
Problemas con la visualización correcta del mapa

### ISSUE THREAD

**Issue by roadev** [pipevel/lapapaya.org#123]
Problema descrito aquí:
http://stackoverflow.com/questions/31887084/rails-gmaps4rails-map-doesnt-autoadjust

**Comment by roadev** [comment]
Probar en vista diferente a userpanel.

**Comment by pipevel** [comment]
No te han respondido nada en stackoverflow?

2015-08-10 23:57 GMT-05:00 Juan Roa <notifications@github.com>:

> Probar en vista diferente a userpanel.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pipevel/lapapaya.org/issues/123#issuecomment-129705882>
> .
>



-- 
Lapapaya.org

!Vitamina para tus sueños!
Blog.lapapaya.org
www.lapapaya.org

Felipe Velasquez Palacio
General Manager
Phone. 3207217643 / 3175658615
Cali - Colombia.

"Haciendo realidad los sueños de los innovadores sociales del Mundo Entero"

**Comment by roadev** [comment]
Sí. Estoy probando en otra vista porque me dijeron que debería ser el css pero ya probé por ahí y nada...
El mapa estático sí funciona (pero es sólo una imágen)
Me dijeron que podría ser algo de la aplicación también...
